Example/PubNub Example.xcodeproj/project.pbxproj

```diff
@@ -501,7 +501,9 @@
 				CLANG_ENABLE_MODULES = YES;
 				CLANG_ENABLE_OBJC_ARC = YES;
 				CLANG_WARN_ASSIGN_ENUM = YES;
+				CLANG_WARN_BLOCK_CAPTURE_AUTORELEASING = YES;
```

Comment:
Why these changes to the .pbxproj file?

Reply:
Well, I've tried to apply suggested configuration, partly managed, but Xcode still suggest to change a lot about build process and architectures.